### PR TITLE
*: cut 2.7.0-rc.1

### PR DESCRIPTION
**Pull request by gouthamve**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 2.7.0-rc.1 / 2019-01-21
+
+We're rolling back the Dockerfile changes introduced in 2.6.0. If you made changes to your docker deployment in 2.6.0, you will need to roll them back.
+
+* [CHANGE] Rollback Dockerfile to version at 2.5.0. Rollback of the breaking change introduced in 2.6.0. #5122
+* [BUGFIX] Make sure the right header is sent when default CORS is set. #5117
```
Comment (brian-brazil) [inline CHANGELOG.md]: This should not be in the final notes.

**Comment by krasi-georgiev** [comment]
[profiles.zip](https://github.com/prometheus/prometheus/files/2780103/profiles.zip)


some profiles to try and make some sense of the slight increase of the memory usage.

**Comment by krasi-georgiev** [comment]
/benchmark cancel

the dashboard should have enough data now.

**Pull request merged**